scripts/download_technique_animations.py

```diff
@@ -43,7 +43,7 @@ def download_bytes(url: str, filepath: str) -> None:
 def process_filename(filepath: str) -> str:
     """Extract base filename from an animation file path."""
     cleaned_filename = os.path.splitext(os.path.basename(filepath))[0]
-    cleaned_filename = cleaned_filename.strip("0123456789_")
```

Comment:
I think remove the strip and the split, but leave the "lower"

Reply:
Will need to fix "12 hits for separation" in the JSON.

Reply:
As we are keeping lower, it's still possible to have collisions. Track the names that you have seen in a set and skip / error if you see the same name twice.

Reply:
> Will need to fix "12 hits for separation" in the JSON.

Do we actually use hits for separation anywhere? I tried searching in the repo, but didn't found any uses in technique JSONs.

> As we are keeping lower, it's still possible to have collisions. Track the names that you have seen in a set and skip / error if you see the same name twice.

We can also check if the gif file exists in `download_bytes`, the way it was kind of done in my other PR, but without checking the file content: https://github.com/Tuxemon/Tuxemon/pull/1318

One other issue I noticed is that the name of the animation is derived from the name of the gif and for some animations like "Explosion Total" the files have prefixes, for instance `120px-*` and `180px-*`. Not sure if these should be addressed on the Wiki or in the script.

Reply:
> Should we error out completely or just report the animation couldn't be downloaded?

Up to you.

Ideally both? Control this via a const or cli arg.

Reply:
> Do we actually use hits for separation anywhere? 

Not sure, I thought we did.